Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,45 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  padding: 48px;
+  position: relative;
+  width: 250px;
+  
+  &:before {
```

Comment:
use `::before` instead of `:before`. the later one is an old version that might be obsolete in the near future.